packages/smooth_app/lib/pages/preferences/user_preferences_page.dart

```diff
@@ -38,7 +39,11 @@ class UserPreferencesPage extends StatefulWidget {
   State<UserPreferencesPage> createState() => _UserPreferencesPageState();
 }
 
-class _UserPreferencesPageState extends State<UserPreferencesPage> {
+class _UserPreferencesPageState extends State<UserPreferencesPage>
+    with TraceableClientMixin {
+  @override
+  String get traceTitle => 'user_preferences_page';
```

Comment:
Probably not 100% relevant btw, as AFAIK this page is always open (as part of the 3 initial tabs).